docs/ci-services.md

```diff
@@ -0,0 +1,55 @@
+# CI Services
+
+The CI service blob defines the supported CI services the user will be
```

Comment:
```suggestion
The CI service map? defines the supported CI services the user will be
```

i'm not sure that "blob" suggests the intended structure to me for this. not that its wrong, but it makes my brain pause. im not sure that map is better. object is probably most technically correct for js, but is probably less informative. what do you think is most clear?